### PR TITLE
Updated README for cartridge installation into conda PostgreSQL

### PR DESCRIPTION
**Pull request by ptosco**
I added a paragraph for each platform dealing with the installation of the PostgreSQL cartridge into `conda` PostgreSQL.

### CODE REVIEW

File: Code/PgSQL/rdkit/README.md [hunk 1]
```diff
@@ -116,6 +154,60 @@ The RDKit cartridge is now to ready to be used on the `nci5K` database.
...
+To test the RDKit cartridge, I create a a small test database:
+```
+>createdb -p 6433 nci5K
+```
+And attempt to load the cartridge in the database:
+```
+>psql -c 'create extension rdkit' -p 6433 nci5k
+CREATE EXTENSION
+```
+Should you see an error mesage such as
```
Comment (rvianello) [inline Code/PgSQL/rdkit/README.md:196]: small typo (message).

Comment (ptosco) [reply Code/PgSQL/rdkit/README.md]: @rvianello Thanks for spotting that! Fixed.

**Review by greglandrum — APPROVED** [review]
LGTM

**Comment by rccc** [comment]
Hello,

I am struggling for days to install it in already installed postgresql.

Documentation is only for conda postgres install only.

We need a documentation for rdkit installed via pip, and a documention for conda rd-hit-postgres for an *existing* database.

I cannot believe i have to spend such time to install the cartbridhe (without success) meanwhile installing Bingo is so easy.

**Comment by greglandrum** [comment]
@rccc This is a complex project with a lot of options and configuration possibilities. And we are all volunteers. The documentation is not perfect and doesn't cover every case. We would all be very happy if that were not the case, but it is never going to be so. 

If there's something which you would like to be able to do and that you can't find documentation for, you have a couple possibilities:
1. Figure out how to do it yourself based on the information which is available and then write the documentation yourself and contribute it to the RDKit. This is how open source projects work.
2. Ask specific question, politely, on the discussion tab in github. Maybe you will get an answer, maybe you won't... there's no guarantee. Again, we're all volunteers doing this because we enjoy it and want to help others. If you are aggressive or rude in how you ask questions you should assume that people are going to be less willing to volunteer their time to help you.

**Comment by rccc** [comment]
Dear @greglandrum 

I have already ask here, and the only time i have a response is when ... i am a quite rude ! 

Now you will understand why, and, at least you give me an answer.


Eric

**Comment by greglandrum** [comment]
@rccc I can only emphasize that you are dealing with human beings who are doing this because they enjoy doing it. Being intentionally rude is also incompatible with the friendly and open nature of this community. 

If you don’t like that friendliness or don’t feel like putting in the effort to not be rude, feel free to look for another project where they welcome that behavior. We don’t.

If you are rude to contributors, they will stop answering you or helping you. And if it gets bad I will, as the admin of the project, take whatever action GitHub allows me to take to solve the problem.

**Comment by rccc** [comment]
:) 

I am not fluent in english, to be rude is not a way to have an answer for me, it is just i am upset.

I understand you spend free time and i had said thank you for this

But just because you threat to take action, please just do it now ! i am not of this kind of scientist  that is afraid by few words or by any actions.

I will not spend my time on this project anyway.

Best regards.

**Pull request merged**